docs/docs/guides/deployment/keyfactor/signserver.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Use Keyfactor SignServer for attestation signing
+---
+
+### Summary
+Chainloop CLI can sign attestations using a preconfigured SignServer instance, by providing a key reference during the signing process:
```

Comment:
I'd also add a paragraph of what Signserver is and a link to their about page https://www.signserver.org/about/